maplibre-adapter/src/androidTest/java/com/amplifyframework/geo/maplibre/MapViewTestActivityTest.kt

```diff
@@ -147,22 +171,22 @@ class MapViewTestActivityTest {
         rule: ActivityScenarioRule<MapViewTestActivity>,
         block: suspend CoroutineScope.() -> T
     ): T {
+        signInWithCognito()
         return runBlocking(Dispatchers.Main) {
             rule.scenario.onActivity {
-                signOutFromCognito() // first sign out to ensure we are in clean state
-                signInWithCognito()
+                it.setMapView(geoCategory)
             }
             block()
         }
     }
 
     private fun signInWithCognito() {
+        signOutFromCognito() // this ensures a previous test failure doesn't impact current test
```

Comment:
Is this necessary since we are creating a new auth plugin for each test? (same comment for other tests)

Reply:
Its for if 1 test fails and sign out doesn't happen, it could cause a cascade of other failures.

Reply:
Ah the plugin instances are distinct but the sharedpreferences persist